src/components/backend-ai-scaling-group-list.ts

```diff
@@ -576,6 +772,56 @@ export default class BackendAIScalingGroupList extends BackendAIPage {
             </mwc-button>
        </div>
       </backend-ai-dialog>
+      <backend-ai-dialog id="modify-scheduler-opts-dialog" fixed backdrop persistent>
+        <span slot="title">${_t('resourceGroup.SetSchedulerOptions')}</span>
+        <div slot="content" id="modify-scheduler-opts-container">
+          <mwc-select
+          id="allowed-session-types"
+          label="allowed_session_types"
+          required>
+            <mwc-list-item style="height:auto;" value="interactive">interactive</mwc-list-item>
```

Comment:
Please check a CSS variable that manages the height of `mwc-list-item`, instead of using inline height in every HTML element.
You can see in [Styled section in README file of mwc-select](https://github.com/material-components/material-web/tree/master/packages/select).

Reply:
I revise it following comment.